Volume/Views/AboutUsView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AboutUsView.swift
+//  Volume
+//
+//  Created by Cameron Russell on 4/12/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct AboutUsView: View {
+    var body: some View {
+        let kvPairs = Constants.subteams.sorted { $0.0.compare($1.0, options: .caseInsensitive) == .orderedAscending }
+        let subteams = kvPairs.map { $0.key }
+        let subteamMembers = kvPairs.map { $0.value }
+        
+        return Group {
+            ScrollView(showsIndicators: false) {
+                VStack(alignment: .leading) {
+                    Header("Our Mission")
+                    Text(Constants.missionStatement)
+                        .font(.helveticaRegular(size: 16))
+                        .lineSpacing(2)
+                        .fixedSize(horizontal: false, vertical: true)
+                        .padding([.bottom, .top])
+                    
+                    Header("The Team")
+                        .padding(.top)
+                    Text(Constants.teamInfo)
+                        .font(.helveticaRegular(size: 16))
+                        .lineSpacing(2)
+                        .fixedSize(horizontal: false, vertical: true)
+                        .padding([.bottom, .top])
+
+                    ForEach(subteams.indices) { index in
```

Comment:
`ForEach(subteams) { subteam in` and then just do `subteam` on line 42 and `kvPairs[subteam]` on line 45.